R/app_ui.R

```diff
@@ -238,25 +238,32 @@ app_ui <- function(request) {
         tabName = "op_am_c2c_referrals",
         mod_mitigators_ui(
           "mitigators_op_c2c_reduction",
-          "Consultant to Consultant Reduction"
+          "Consultant to Consultant Reduction",
+          FALSE
```

Comment:
So we can remember what `FALSE` refers to.

```suggestion
          show_diagnoses_table = FALSE
```